### ISSUE TITLE
Application error

### ISSUE THREAD

**Issue by ThatOneCalculator** [cffnpwr/caffe-bruncher#2]
## Dev:

![image](https://user-images.githubusercontent.com/44733677/180057305-437dd6d4-6322-4522-88d3-d86cb1bba431.png)

```
lib/utils.ts (12:20) @ countGraphemeForTwitter

  10 | 
  11 | export const countGraphemeForTwitter = (str: string) => {
> 12 |   const segmenter = new Intl.Segmenter('ja', { granularity: 'grapheme' });
     |                    ^
  13 |   const partition = devideString(str);
  14 | 
  15 |   return partition.reduce((count: number, text: string): number => {
```

## Production:

![image](https://user-images.githubusercontent.com/44733677/180056960-5918d1fb-0101-45c6-87df-a38d5895edc5.png)

```
TypeError: Intl.Segmenter is not a constructor
    NextJS 11
        I
        G
        Eo
        xi
        bs
        gs
        vs
        as
        ls
        S
        T
[framework-4556c45dd113b893.js:1:71883](http://always.voring.me/_next/static/chunks/framework-4556c45dd113b893.js)
TypeError: Intl.Segmenter is not a constructor
    NextJS 11
[main-ef9379c68abd1065.js:1:8788](http://always.voring.me/_next/static/chunks/main-ef9379c68abd1065.js)
    NextJS 13
A client-side exception has occurred, see here for more info: https://nextjs.org/docs/messages/client-side-exception-occurred [main-ef9379c68abd1065.js:1:8805](http://always.voring.me/_next/static/chunks/main-ef9379c68abd1065.js)
Uncaught Error: Minified React error #423; visit https://reactjs.org/docs/error-decoder.html?invariant=423 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    NextJS 8
[framework-4556c45dd113b893.js:1:121812](http://always.voring.me/_next/static/chunks/framework-4556c45dd113b893.js)
    NextJS 6
        xi
        bs
        gs
        vs
        as
        ls
    ls self-hosted:1162
    NextJS 2
```

**Comment by cffnpwr** [comment]
Probably due to a low version of Node.js.
I think that setting the Node.js version to 18 or higher will solve the problem.

**Comment by ThatOneCalculator** [comment]
My NodeJS version is 18.4.0 though.

**Comment by cffnpwr** [comment]
The error seems to be caused because `Intl.Segmenter` is not implemented in Firefox.
Fixed in ebfb8c7f3d8f109c3dbb6dc6575d7e0c73dbfca2 and 8a1827da1b7af68bc453c6505dd58a3bde3a16e5.
